src/commands/miscellaneous/info.ts

```diff
@@ -54,7 +54,7 @@ const infoCommandDetails: CodeyCommandDetails = {
   subcommandDetails: {}
 };
 
-export class InfoCommand extends CodeyCommand {
+export class MiscellaneousCommand extends CodeyCommand {
```

Comment:
This should be `MiscellaneousInfoCommand`?

Reply:
Oops good catch thanks!